src/up.cmd.js

```diff
@@ -159,20 +159,24 @@ class UpCommand extends BuildCommand {
     }
 
     // check all local repos
-    const localRepos = await Promise.all(this._localRepos.map(async (repo) => {
+    const localRepos = (await Promise.all(this._localRepos.map(async (repo) => {
       const repoPath = path.resolve(this.directory, repo);
       if (!await fse.pathExists(path.join(repoPath, '.git'))) {
         throw Error(`Specified --local-repo = ${repo} is not a git repository.`);
       }
       const gitUrl = await GitUtils.getOriginURL(repoPath);
       if (!gitUrl) {
+        if (repo === '.') {
```

Comment:
this fixes it, but I think the error is a different one, such as the `.` is a new default.
this was changed #913. I think the check for a local repo should be different.

the original idea was that you can specify a local repo that overrides the resolution for git repositories specified in the strains. Before any strains are setup and before the local git has no origin, then the simulator anyways uses the cwd.

so I think that #913 should be fixed in a way, that it sets `.` as default, only if the local repo has an origin.